core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/executionrequest/ExecutionRequestController.java

```diff
@@ -86,7 +86,7 @@ public ExecutionRequestDto post(@RequestBody ExecutionRequestDto executionReques
         for (ExecutionRequestDto executionRequestDto : executionRequestDtos) {
             halMultipleEmbeddedResource.embedResource(executionRequestDto);
             halMultipleEmbeddedResource.add(WebMvcLinkBuilder.linkTo(methodOn(ScriptController.class)
-                    .getByName(executionRequestDto.getName(), null))
+                    .getByName(PageRequest.of(0, 20), executionRequestDto.getName(), null, ""))
```

Comment:
I think it should point to a script and thus a get instead of a getByName which point several version of a script.